scopes/component/ui/component-meta/component-overview.tsx

```diff
@@ -44,30 +53,59 @@ export function ComponentOverview({
     const origin = isBrowser ? window.location.origin : undefined;
     finalElementsUrl = origin && elementsUrl ? `${origin}${elementsUrl}` : undefined;
   }
+
+  type CompWithTitleBadgesProps = { position: TitleBadgePosition; children: ReactNode };
+  function ComponentWithTitleBadges({ position, children }: CompWithTitleBadgesProps) {
+    return (
+      <div className={styles.componentWithBadgeContainer}>
+        {children}
```

Comment:
do we need the container? I think you can just make a Badges component and then use it like so:

```
  <div className={styles.componentTitle}>
    <H1>{displayName}</H1>
      <Badges value={titleBadges} position={TitleBadgePosition.Title}
  </div>
```

Reply:
removed